rom/dev/README.md

```diff
@@ -397,10 +390,8 @@ Local Device ID Layer derives the Owner CDI & ECC Keys. This layer represents th
 
 | Slot | Key Vault | PCR Bank | Data Vault 48 Byte (Sticky) | Data Vault 4 Byte (Sticky) |
 |------|-----------|----------|-----------------------------|----------------------------|
-| 0 | | | 🔒LDevID Pub Key X |
-| 1 | | | 🔒LDevID Pub Key Y |
-| 2 | | | 🔒LDevID Cert Signature R |
```

Comment:
How do we generate cert chains without pub key. I want to make sure all content generated by ROM that is critical for the Caliptra IP to function correctly without depending on DCCM. This change violates this design principle